cypress/locators/drawer/locators.js

```diff
@@ -0,0 +1,5 @@
+// component preview locators
+export const DRAWER = '[data-element="drawer"]';
+export const DRAWER_TOGGLE = '[data-element="drawer-toggle"]';
+export const DRAWER_SIDEBAR = '[data-element="drawer"] div:nth-child(1)';
+export const DRAWER_SIDEBAR_CONTENT = `${DRAWER_SIDEBAR} > div`;
```

Comment:
move it to index.js file as
`export const drawerSidebarContent = () => drawerSidebar().find('div')`

Reply:
fixed 4d965d0